src/structures/GuildMember.js

```diff
@@ -284,6 +302,18 @@ class GuildMember extends Base {
     return this.edit({ nick }, reason);
   }
 
+  
+  /**
+   * Disables or enables communication for this member.
+   * @param {?Date} The duration in milliseconds the guild member should have communication disabled for, or `null` to enable communication for them
+   * @param {string} [reason] Reason for setting the nickname
+   * @returns {Promise<GuildMember>}
+   */
+  disableCommunication(duration, reason) {
+    const timestamp = Date.now()+duration;
+    return this.edit({communication_disabled_until: new Date(timestamp).toString()}, reason);
```

Comment:
I highly doubt discord would require you to send an ISO string to modify a guild member, it's likely to be a value in seconds, but this still needs to be added to the #edit method (also formatting)